### DISCUSSION TITLE
May I run onedrive (private) and onedrive for business at the same time and syncing in different directories

### DISCUSSION THREAD

**Discussion by MJedamzik** [abraunegg/onedrive#3024] [General]
Hi there,

I have a "onedrive" for private use and my employer offers a "onedrive for business". I like to sync both with my Ubuntu 24.04 LTS System. I have different logins and I´d like to sync in different folders (apart form the standard ~/onedrive/).

Would that be actually possible? Is it hidden somewhere in the docs, as I couldnt find it yet.

Thanks in advance for your help.

**Comment by abraunegg** [comment]
@MJedamzik 
> I have a "onedrive" for private use and my employer offers a "onedrive for business". I like to sync both .... I have different logins and I´d like to sync in different folders (apart form the standard ~/onedrive/).
> Would that be actually possible? Is it hidden somewhere in the docs, as I couldnt find it yet.

This is entirely possible to achieve - a high level outline of what you need to do is below:
1. Configure the application to use 'configuration directories' so that you are running 2 x clients with different configurations. Please read:
  - https://github.com/abraunegg/onedrive/blob/master/docs/usage.md#advanced-configuration-of-the-onedrive-client-for-linux
  - https://github.com/abraunegg/onedrive/blob/master/docs/advanced-usage.md#configuring-the-client-to-use-multiple-onedrive-accounts--configurations
2. To advise the client where to store your data in different folders (from the standard ~/OneDrive) you need to configure the 'sync_dir' option which is part of the details above:
  - https://github.com/abraunegg/onedrive/blob/master/docs/application-config-options.md#sync_dir

> with my Ubuntu 24.04 LTS System

You have chosen a poor quality distribution to use, and LTS really should be renamed Long Term Stuffed as bugs in other packages that this client depend on are not fixed by Ubuntu. You will have to do some platform remediation (or use a better distribution). Please read:
* https://github.com/abraunegg/onedrive/blob/master/docs/usage.md#compatibility-with-curl
* https://github.com/abraunegg/onedrive/discussions/2997
